clerk/organization_roles.go

```diff
@@ -0,0 +1,123 @@
+package clerk
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+)
+
+type OrganizationRolesService service
+
+type OrganizationRole struct {
```

Comment:
Do we want to have separate types for each role context or make it a generic one `Role`? Whatever we think is the best, let's make sure to align the `Permission` struct name as well

Reply:
Let’s keep it separate for now - I think it’s easier to understand this way what the struct is for!